packages/legend-application-studio/src/components/editor/edit-panel/connection-editor/RelationalDatabaseConnectionEditor.tsx

```diff
@@ -1151,6 +1137,217 @@ const RelationalConnectionStoreEditor = observer(
   },
 );
 
+const renderEditorPostProcessor = (
+  connectionValueState: RelationalDatabaseConnectionValueState,
+  postprocessor: PostProcessor,
+  plugins: LegendStudioApplicationPlugin[],
+): React.ReactNode => {
+  if (postprocessor instanceof MapperPostProcessor) {
+    return (
+      <MapperPostProcessorEditor
+        connectionValueState={connectionValueState}
+        postprocessor={postprocessor}
+      />
+    );
+  } else {
+    const extraPostProcessorEditorRenderers = plugins.flatMap(
+      (plugin) =>
+        (
+          plugin as StoreRelational_LegendStudioApplicationPlugin_Extension
+        ).getExtraPostProcessorEditorRenderers?.() ?? [],
+    );
+    for (const editorRenderer of extraPostProcessorEditorRenderers) {
+      const editor = editorRenderer(postprocessor, connectionValueState, false);
+      if (editor) {
+        return editor;
+      }
+    }
+    return (
+      <UnsupportedEditorPanel
+        isReadOnly={true}
+        text={`Can't display post processor in form mode`}
+      ></UnsupportedEditorPanel>
+    );
+  }
+};
+
+const PostProcessorRelationalConnectionEditor = observer(
+  (props: { connectionValueState: RelationalDatabaseConnectionValueState }) => {
+    const { connectionValueState } = props;
+
+    const connection = connectionValueState.connection;
+
+    const postprocessors = connection.postProcessors;
+
+    const editorStore = useEditorStore();
+    const observerContext = editorStore.changeDetectionState.observerContext;
+    const plugins = editorStore.pluginManager.getApplicationPlugins();
+
+    const selectedPostProcessor = connectionValueState.selectedPostProcessor;
+
+    const deletePostProcessor =
+      (postprocessor: PostProcessor): (() => void) =>
+      (): void => {
+        postProcessor_deletePostProcessor(connectionValueState, postprocessor);
+        if (postprocessor === connectionValueState.selectedPostProcessor) {
+          connectionValueState.setSelectedPostProcessor(undefined);
+        }
+      };
+    const postprocessorOptions = (
```

Comment:
Let's check and make sure we capitalize properly everywhere, i.e. `postProcessor`. Thanks!